assets/js/components/BadgeWithTooltip.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Audience Segmentation PartialDataBadge component.
+ * Audience Segmentation BadgeWithTooltip component.
```

Comment:
This is no longer an Audience Segmentation specific component as this has been moved to `assets/js/components`.

```suggestion
 * BadgeWithTooltip component.
```